components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -163,6 +166,12 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('tournament_participant_number', 0)
 	Variables.varDefine('tournament_participants', '(')
 	Variables.varDefine('tournament_teamplayers', args.mode == _MODE_2v2 and 2 or 3)
+	Variables.varDefine('showh2h',
+		Logic.emptyOr(
+			args.showh2h,
+			tostring(tonumber(args.liquipediatier or 0) <= _H2H_TIER_THRESHOLD)
```

Comment:
Are you sure you want to default to `0` on missing/invalid tiers?

Reply:
You're right that it probably does make more sense to hide H2H for invalid tiers, but I couldn't think of a pretty way in code to put an arbitrarily large tier so I just went with 0. Is there some kind of `MAX_TIER` constant anywhere?